OptimoveSDK/optimove-sdk/src/main/java/com/optimove/android/Optimove.java

```diff
@@ -115,6 +115,8 @@ public static Optimove getInstance() {
     public static void initialize(@NonNull Application application, @NonNull OptimoveConfig config) {
         currentConfig = config;
 
+        performSingletonInitialization(application.getApplicationContext(), config);
```

Comment:
this is not guaranteed to run on main thread anymore. Was it important?

Reply:
This shouldnt be important anymore